[hikogui] update to version 0.7.0

takev opened the pull request:
Update hikogui to version 0.7.0.
This library was called ttauri, which is a port in vcpkg already, but was renamed.

The port files where simply copied and modified for the new version, therefor the port file may have old statements in them.

- #### What does your PR fix?
no fixes.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
no changes: x64-windows-static

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
Yes.

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

takev committed:
[hikogui] update to version 0.7.0

JonLiu1993 commented:
@takev, thanks for the PR, could you update ttauri instead of adding a new port of hikogui?

dg0yt commented:
> @takev, thanks for the PR, could you update ttauri instead of adding a new port of hikogui?

We need both ports: `hikogui` as proposed, and `ttauri` as an empty port depending on `hikogui`. Cf. `proj` vs. `prj4`.

takev committed:
[hikogui] empty ttauri and fix deprecated port files.

Make ttauri an empty dependency only port which imports
hikogui.

Update the hikogui portfile to use non-deprecated functions.

takev commented:
@dg0yt I have updated the pull request to make ttauri empty and depended on hikogui.
It does not make much sense to create polyfills, as many things have changed. Also both ttauri and hikogui are still pre-version-1.

takev committed:
[ttauri] version update

JonLiu1993 commented on ports/hikogui/vcpkg.json:
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "hikogui",
+  "version": "0.7.0",
+  "port-version": 2,
```
```suggestion
```

JonLiu1993 replied:
We should remove `port-version` when we add a new port

JonLiu1993 commented on versions/h-/hikogui.json:
```diff
@@ -0,0 +1,14 @@
+      "version": "0.7.0",
+      "port-version": 2
+    },
+    {
+      "git-tree": "708ad193cf5264aeb0738c9e12069bae5c816597",
+      "version": "0.7.0",
+      "port-version": 1
+    }
+  ]
```
```suggestion
```

JonLiu1993 replied:
We just need to add one git-tree record.

takev committed:
Update ports/hikogui/vcpkg.json

Co-authored-by: JonLiu1993 <63675417+JonLiu1993@users.noreply.github.com>

JonLiu1993 commented on versions/h-/hikogui.json:
```diff
@@ -0,0 +1,14 @@
+{
+  "versions": [
+    {
+      "git-tree": "6d88971773847339a04ec50311d8b40eaf0f7982",
+      "version": "0.7.0",
+      "port-version": 2
+    },
+    {
+      "git-tree": "708ad193cf5264aeb0738c9e12069bae5c816597",
+      "version": "0.7.0",
+      "port-version": 1
+    }
+  ]
```
```suggestion
```

JonLiu1993 replied:
Please remove this code and run command "./vcpkg x-add-version --all --overwrite-version" and commit again.

takev replied:
Done.

takev committed:
remove port-version
